test/acceptance/tests.py

```diff
@@ -35,7 +35,7 @@ def _inner(*args, **kwargs):
                     return func(*args, **kwargs)
                 except (BrokenPromise, AssertionError) as ex:
                     if attempt_num >= (tries - 1):
-                        raise ex
+                        raise
```

Comment:
This will preserve the stack trace on failed test runs, which was extra helpful in writing these